src/App.js

```diff
@@ -1,24 +1,35 @@
 import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import NavBar from './components/NavBar/NavBar';
+import Users from './components/UserProfile/UserProfile';
+import Posts from './components/Posts/Posts';
+import Contact from './components/Contact/Contact'
+// import logo from './logo.svg';
+// import './App.css';
 
-function App() {
+const App = () => {
   return (
     <div className="App">
       <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
+       <NavBar />
       </header>
+      
+        <div>
+          <Users />
+        </div>
+        <div>
+        <h3>Posts</h3>
+          <Posts posts = {[{title: "Title", description: "Loren Ipsem" },
```

Comment:
to keep these cleaner you can keep them in a variable  right below state and pass it here later. similar how we will use state